tests/test-sources/plugins/by-name/rust-tools/default.nix

```diff
@@ -1,11 +1,13 @@
 {
   empty = {
-    plugins.rust-tools.enable = true;
+    # Plugin deprecated
+    # plugins.rust-tools.enable = true;
```

Comment:
Instead of disabling the test, can we instead disable the test checking warnings?

`test.checkWarnings = false` iirc

In an ideal world, we'd "expect" a specific warning to test _that_ works, but we don't have that implemented yet in the test module.

Reply:
Ah I forgot about that functionality